bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -854,7 +894,8 @@ func (r *KubeadmConfigReconciler) storeBootstrapData(ctx context.Context, scope
 			},
 		},
 		Data: map[string][]byte{
-			"value": data,
+			"value":  data,
+			"format": []byte(bootstrapDataFormat(scope.Config.Spec)),
```

Comment:
Should this be an annotation instead?

Reply:
>Should this be an annotation instead?

I guess either will be OK. Is there some argument to use annotation over field here? That it's not really a `Data`?

Reply:
@vincepri ?